frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -0,0 +1,72 @@
+import { useOktaAuth } from "@okta/okta-react";
+import { useState } from "react";
+import { permissionCheck, reportReceiver } from "../../webreceiver-utils";
+import { PERMISSIONS } from "../../resources/PermissionsResource";
+
+import { Header, Title, PrimaryNav, Link, NavMenuButton } from "@trussworks/react-uswds";
+import { OrganizationDropdown } from './OrgDropdown'
+import { SignInOrUser } from "./SignInOrUser";
+import { HIWDropdown } from "./HIWDropdown";
+
+export const ReportStreamHeader = () => {
+    const { authState } = useOktaAuth();
+    const [expanded, setExpanded] = useState(false)
+    const toggleMobileNav = (): void => setExpanded((prvExpanded) => !prvExpanded)
+    let itemsMenu = [
+        <Link href="/about" id="docs" className="usa-nav__link">
+            <span>About</span>
+        </Link>,
+        <HIWDropdown />,
+    ];
+
+    if (authState !== null && authState.isAuthenticated) {
+        if (reportReceiver(authState)) {
+            itemsMenu.splice(0, 0,
+                <Link href="/daily-data"
+                    key="daily"
+                    data-attribute="hidden"
+                    hidden={true}
+                    className="usa-nav__link">
+                    <span>Daily data</span>
+                </Link>
+            );
+        }
+
+        if (permissionCheck(PERMISSIONS.SENDER, authState)) {
+            itemsMenu.splice(1, 0,
+                <Link href="/upload"
+                    key="upload"
+                    data-attribute="hidden"
+                    hidden={true}
+                    className="usa-nav__link">
+                    <span>Upload</span>
+                </Link>
```

Comment:
These need to be NavLinks now! if you rebase and re-copy-and-paste you should be good

Reply:
Will do!